docs/DeveloperGuide.md

```diff
@@ -2,24 +2,138 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Add Function
 
+#### System Architecture of the Add Function<br/>
+
+![](team/Architect-digram.PNG)
+
+The Architecture Diagram given above explains the high-level design of the add command.<br/>
+
+Below we will discuss a quick overview of each component.
+
+* UI: The User Interface of the app<br/>
+* Add Command: The main logic command of the add function<br/>
+* Timetable: The arraylist where events that are added are stored accordingly<br/>
+
+#### Design of the Add Function<br/>
+
+UML Class Diagram of the Add function:
+![](team/uml.PNG)<br/>
+
+1) The Add function gets the inputs from the User and places it to the event class so that 
+it can parser to string.<br/>
+2)  It will then pass the event data onto UI class to display and inform user of the event added 
+to the timetable.<br/>
+3) Next,the add function will pass all the parameters (event name,day,time
+and location) to the timetable class into its arraylist divided by the day of the week<br/>
+
+#### Sequence Design of Add Function<br/>
+![](team/seqdiagram.jpg)<br/>
```

Comment:
![image](https://user-images.githubusercontent.com/53657436/97386637-350a3f80-190f-11eb-81aa-6903024aa521.png)

You could perhaps indicate more clearly where the arrows are coming from or going.
For the sake of neatness you could try to do the diagrams in software like powerpoint some or uml creator or something instead of writing cause the pencil marks here are a bit messy sorry

Reply:
You could also perhaps use the same software to create your uml diagrams for the sake of consistency, as the diagrams vary in style along the document.

Reply:
![image](https://user-images.githubusercontent.com/60027323/97515331-cf2fbd80-19cb-11eb-9874-45422dfea44f.png)
1. parsedInputs[] cannot be a method call. 
2. "Return" lines should orginate from the point of deactivation
3. "new event" method call line should not point backwards in time. Perhaps you can reformat this diagram such that it is a straight line.